account_loan/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Account Loan management
+=======================
+
+This module extends the functionality of accounting to support loans.
+It will create automatically moves or invoices for loans.
+Moreover, you can check the pending amount to be paid and reduce the debt.
+
+It currently supports two kinds of debts:
+
+* Loans: a standard debt with banks, that only creates account moves
+* Leases: a debt with a bank where purchase invoices are necessary
```

Comment:
I do not see the estimated residual value of the asset after the lease period. I understand that sometimes will want to lease an equipment until it reaches a certain residual value, in which you might want to exercise the right to purchase the asset.

Reply:
Added the option of residual value on leases (not on loans, but it could be added).
Maybe it would be good to add a button to create this move automatically on the end, or do you thins it is better to leave the information in order to create the move manually?